docs/guides/ce-resource-management.rst

```diff
@@ -0,0 +1,96 @@
+Compute Engine Resource Management
+=======================
+
+Workflows submitted to run on Compute Engine can specify the computational resources they require, enabling precise management of resources and costs.
+
+.. note::
+    Resource management is supported only when executing on Compute Engine runtimes. Tasks and workflows defined with these parameters may still be executed on other runtimes, however the resource specification will be ignored.
+
+Setting Task Resources
+----------------------
+
+Required hardware resources are configured on a per-task basis by setting the ``resources`` field of the task decorator:
+
+.. code-block::
+   :caption: task resource request example
+
+    @sdk.task(
+        resources=sdk.Resources(cpu="100m", memory="1Gi", disk="10Gi", gpu="1")
+    )
+    def my_task():
+        ...
+
+``resources`` expects a ``sdk.Resources()`` object that specifies some or all of:
+* ``cpu``: number of cores.
+* ``memory``: amount of RAM (bytes).
+* ``disk``: disk space (bytes).
+* ``gpu``: whether access to a gpu unit is required (``1`` if a GPU is required, ``0`` otherwise).
+
+Amounts of cpu and memory resources can be specified as a plain integer, or as a fixed-point float appended with the string representation of one of the following unit prefixes. These can be SI (metric), or IEC (binary) prefixes. The specified prefix will be used to modify the base unit for the parameter, 'byte' in the case of ``memory`` and ``disk`` requests, 'cores' in the case of ``cpu`` requests. So ``disk="10k"`` will be interpreted as '10 kilobytes', while ``cpu="10k"`` would request 10^7 cores.
```

Comment:
Suffixes, not prefixes! I made the same mistake the other day 😅

Reply:
Okay so this is definitely something we need to be careful about the language on. They're prefixes w.r.t. the unit, but suffixes w.r.t. the actual string you write. 'Prefixes' makes sense to me, but as you point out are likely to cause confusion, so we should avoid using the word.
What do you think about:

```suggestion
Amounts of cpu and memory resources are specified by a string comprising a floating point value, and, optionally, a modifier to the base unit ('byte' in the case of ``memory`` and ``disk`` requests, 'cores' in the case of ``cpu`` requests). The modifier can be a SI (metric), or IEC (binary) multiplier as detailed in the table below. So ``disk="10k"`` will be interpreted as '10 kilobytes', while ``cpu="10k"`` would request 10^7 cores.
```

Reply:
Ahh yeah you're right! suffix to the "number" but prefix to the "unit".

I think your suggestion is good.